src/main/scala/tilelink/ToAXI4.scala

```diff
@@ -105,15 +110,17 @@ class TLToAXI4(val combinational: Boolean = true, val adapterName: Option[String
       // Construct the source=>ID mapping table
       val map = new TLtoAXI4IdMap(edgeIn.client)
       val sourceStall = Wire(Vec(edgeIn.client.endSourceId, Bool()))
-      val sourceTable = Wire(Vec(edgeIn.client.endSourceId, out.aw.bits.id))
-      val idStall = Wire(init = Vec.fill(edgeOut.master.endId) { Bool(false) })
+      sourceStall.foreach(_ := false.B)
```

Comment:
why tie off `sourceStall` by default?

Reply:
In the foreach loop a few lines below, `tlId.size` could be smaller than `edgeIn.client.endSourceId`, causing some values to be uninitliazed, thus regression run 3 to fail